src/lifedrain.py

```diff
@@ -48,6 +49,24 @@ def __init__(self, mw: AnkiQt, qt: Any):
             'card_type': None,
         }
 
+    def reset_daily(self) -> None:
+        """Resets the lifedrain bar for all decks once per day.
+
+        Uses Anki's rollover hour instead of a hard-coded 4am.
+        """
+        # Safely get the rollover hour from Anki's config.
+        # Default to 4 if it's not set.
+        if self._mw.col is None:
+            exp = 'Collection is not loaded'
+            raise RuntimeError(exp)
```

Comment:
Not sure if raising an exception here is a good idea. It stops the add-on from working, and people may come here asking for a fix.

I think we can just `return` instead.